common/source/docs/common-scripting-aerobatics.rst

```diff
@@ -0,0 +1,109 @@
+.. _common-scripted-aerobatics:
+
+===================
+Scripted Aerobatics
+===================
+
+
+.. note:: this capability is in ArduPlane 4.2 and higher
+
+ArduPilot has the capability of executing aerobatics from a LUA script. Either via AUTO mode mission items, or via scripts executed in many normal plane flight modes, usually controlled via a transmitter switch and selection RC channels.
+
+Mission NAV_SCRIPT_TIME 
+=======================
+
+The SCRIPT_TIME waypoint type allows for a :ref:`LUA<common-lua-scripts>` script to take control of the roll/pitch/yaw rates of the vehicle.  The mandatory timeout parameter is for protection against the lack of an appropriate script being active (ie was not loaded in the APM/Scripts directory of the SD card), or LUA script crash due to error.  The timeout value specifies the maximum time the script can take over before the next mission item is executed.
+
+The format of the SCRIPT_TIME item is:
+
+.. raw:: html
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Command Field</th>
+   <th>Mission Planner Field</th>
+   <th>Description</th>
+   </tr>
+   <tr>
+   <td><strong>param1</strong></td>
+   <td>
+   </td>
+   <td>Command ID number, accessible from running script to identify requested maneuver</td>
+   </tr>
+   <tr>
+   <td><strong>param2</strong></td>
+   <td></td>
+   <td>Maximu time in seconds before next mission item is processed</td>
+   </tr>
+   <tr>
+   <td><strong>param3</strong></td>
+   <td></td>
+   <td>Parameter which is accessible from running script for maneuver use a first variable.</td>
+   </tr>
+   <tr>
+   <td><strong>param4</strong></td>
+   <td></td>
+   <td>Parameter which is accessible from running script for maneuver use as a second variable..</td>
+   </tr>
+   <tr style="color: #c0c0c0">
+   <td><strong>param5</strong></td>
+   <td>Lat</td>
+   <td>Target latitude</td>
+   </tr>
+   <tr style="color: #c0c0c0">
+   <td><strong>param6</strong></td>
+   <td>Lon</td>
+   <td>Target longitude</td>
+   </tr>
+   <tr style="color: #c0c0c0">
+   <td><strong>param7</strong></td>
+   <td>Alt</td>
+   <td>Target altitude</td>
+   </tr>
+   </tbody>
+   </table>
+
+A LUA script would need to be running when the above mission command is encountered to recognize that it needs to take control, and start sending pitch/roll/yaw rate inputs into the flight controller to control its attitude and/or trajectory. It can use the two variable input parameters of the command to control aspects of the maneuver, such as rate of rolls/loops, repeat counts, time for certain segments of the maneuver, etc. An example is provided in the LUA Scripting Examples directory called `plane_aerobatics.lua </home/henry/Desktop/ardupilot/libraries/AP_Scripting/examples/Aerobatics/Missions>`__ for several types of tricks.
```

Comment:
bad link in here, /home/henry/blah

Reply:
Can't put correct link until my PR gets merged moving that script